roles/bind-keycloak-apb/tasks/main.yml

```diff
@@ -15,7 +15,7 @@
 
 - name: "Generate keycloak auth token"
   uri:
-    url: "http://{{ keycloak_route.stdout }}/auth/realms/master/protocol/openid-connect/token"
+    url: "https://{{ keycloak_route.stdout }}/auth/realms/master/protocol/openid-connect/token"
```

Comment:
can we make the protocol value a parameter and default it to `https`? I do think there might be a use case of http for testing purpose only.

@aidenkeating WDYT?

Reply:
@wei-lee Yeah that's a good point 👍